<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creation_automate</class>
 <widget class="QDialog" name="creation_automate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>485</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nouvel Automate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="4">
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Automate 1D</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jeu de la Vie</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Autres Automates 2D</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QComboBox" name="comboBox_3">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Torique</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="14" column="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QSpinBox" name="spinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Effet de Bord :</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>14</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>14</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>39</number>
     </attribute>
     <row/>
     <column/>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>14</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>14</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>39</number>
     </attribute>
     <row/>
     <column/>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Nombre de colonnes :  </string>
     </property>
    </widget>
   </item>
   <item row="10" column="4">
    <widget class="QSpinBox" name="spinBox_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QSpinBox" name="spinBox_6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Nombre d'états :  </string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="5">
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>14</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>14</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>39</number>
     </attribute>
     <row/>
     <column/>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QComboBox" name="comboBox_2">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Aléatoire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Aléatoire Simétrique</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Personalisée</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Génération :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Type d'Automate :</string>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QSpinBox" name="spinBox_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_9">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Numéro Automate 1D :</string>
     </property>
    </widget>
   </item>
   <item row="11" column="4">
    <widget class="QSpinBox" name="spinBox_5">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>127</number>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_10">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Numéro Automate 1D Binaire :</string>
     </property>
    </widget>
   </item>
   <item row="12" column="4">
    <widget class="QLineEdit" name="lineEdit">
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>01111111</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Min Cellules :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Max Cellules : </string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Nombre de lignes :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QSpinBox" name="spinBox_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
